src/storage/hummock_test/src/compactor_tests.rs

```diff
@@ -599,7 +648,7 @@ pub(crate) mod tests {
         );
 
         let compact_ctx = get_compactor_context_with_filter_key_extractor_manager(
-            &storage,
+            &storage_1,
```

Comment:
Yes, its confusing，i made some changes.
1. In this pr , using HummockV2MixedStateStore means that we register a table_id and instance to `global_state_store`, so that we can use `global_state_store` for batch scan. (add some doc)
2. compact_ctx is only not rely on input `storage` directly, so i modified the incoming parameter (storage_1) to avoid misunderstanding
3. use `global_storage` for scan results instead of local_storage.